src/Entity/EventCategory.php

```diff
@@ -22,4 +22,19 @@
  */
 class EventCategory extends BaseEventCategory
 {
+    /**
+     * @ORM\ManyToOne(targetEntity="AppBundle\Entity\EventGroupCategory", inversedBy="eventCategories")
+     * @ORM\JoinColumn(name="event_group_category_id", referencedColumnName="id", nullable=true)
```

Comment:
I think you can avoid this annotation, by default the column will be nullable

Reply:
removed